src/pg/replication.cr

```diff
@@ -0,0 +1,353 @@
+module PG::Replication
+  module Handler
+    def received(frame)
+    end
```

Comment:
This is needed so that you don't get compile-time errors when you don't define all possible messages. It doesn't have to be untyped, though. It might be better to be explicit that this is for `WALMessage` objects only.